engine/flink/process/src/test/scala/pl/touk/nussknacker/engine/process/FlinkStreamingProcessRegistrarKafkaSpec.scala

```diff
@@ -47,7 +46,7 @@ class FlinkStreamingProcessRegistrarKafkaSpec
         .processorEnd("service", "logService", "all" -> "#input"))
     )
 
-    ProcessTestHelpers.processInvoker.invokeWithKafka(
+    helpers.ProcessTestHelpers.processInvoker.invokeWithKafka(
```

Comment:
do we need that change actually?

Reply:
we don't but it removes compiler warning